taxi_service/urls.py

```diff
@@ -22,4 +22,5 @@
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("", include("taxi.urls", namespace="taxi")),
```

Comment:
The URL identification parameter `namespace` should use `-` instead of `_` according to the checklist. Consider changing `namespace="taxi"` to `namespace="taxi-service"`.